android/src/main/java/com/xmartlabs/rnline/RNLine.kt

```diff
@@ -233,7 +232,7 @@ class RNLine(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule
             )
     )
 
-    private fun parseVerifyAccessToken(verifyAccessToken: LineCredential): WritableMap = Arguments.makeNativeMap(
+    private fun parseVerifyAccessToken( verifyAccessToken: LineCredential): WritableMap = Arguments.makeNativeMap(
```

Comment:
```suggestion
    private fun parseVerifyAccessToken(verifyAccessToken: LineCredential): WritableMap = Arguments.makeNativeMap(
```
You left a whitespace there 😛